### PR TITLE
Changed mom_saveloc_current

### LINKED ISSUE #248: document +mom_saveloc_current
https://github.com/momentum-mod/game/issues/1337 is in, so there should be a note about it on the `mom_saveloc_current` page. 

By using `+mom_saveloc_current` instead, there is the functionality to hold down the key to load the saveloc and freeze in place, then release key to unfreeze.

### PR DESCRIPTION
**Pull request by OfficialDecoy**
Closes #248

### CODE REVIEW

File: _posts/commands/2020-05-01-mom_saveloc_current.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ tags:
 safeguard: mom_run_safeguard_saveloc_tele
 ---
 
-Teleports the player to their current saved location.
\ No newline at end of file
+Teleports the player to their current saved location. If player holds down the key it will load the saveloc and freeze the player in place, releasing the key will unfreeze the player.
```
Comment (braem) [inline _posts/commands/2020-05-01-mom_saveloc_current.md:10]: This behaviour works only if `+mom_saveloc_current` is bound, not just `mom_saveloc_current`. (`+mom_saveloc_current` loads the saveloc and freezes in place, and `-mom_saveloc_current` unfreezes player).

**Review by braem — CHANGES_REQUESTED** [review]

File: _posts/commands/2020-05-01-mom_saveloc_current.md [hunk 2]
```diff
@@ -7,4 +7,4 @@ tags:
 safeguard: mom_run_safeguard_saveloc_tele
 ---
 
-Teleports the player to their current saved location.
\ No newline at end of file
+Teleports the player to their current saved location. Binding +mom_saveloc_current and holding the set key down, freezes the player and loads the saveloc. Upon releasing the set key (-mom_saveloc_current) will unfreeze the player.
```
Comment (braem) [inline _posts/commands/2020-05-01-mom_saveloc_current.md:10]: Verryy minor nitpick, last one i promise!
```suggestion
Teleports the player to their current saved location. Binding +mom_saveloc_current and holding the set key down, freezes the player and loads the saveloc. Releasing the set key (-mom_saveloc_current) will unfreeze the player.
```

**Review by braem — CHANGES_REQUESTED** [review]

File: _posts/commands/2020-05-01-mom_saveloc_current.md (hunk 2 above)
Comment (OfficialDecoy) [reply _posts/commands/2020-05-01-mom_saveloc_current.md:10]: The "And" in red, you want them removed?

**Review by braem — APPROVED** [review]
Nice work :D 
DM me on Discord (brae#0047) if you want a game key and don't already have one :p

File: _posts/commands/2020-05-01-mom_saveloc_current.md (hunk 2 above)
Comment (braem) [reply _posts/commands/2020-05-01-mom_saveloc_current.md]: I think it's coloring that like it's some kind of programming logic haha. DW about it

**Pull request merged**